src/Struct/FindologicProduct.php

```diff
@@ -628,6 +628,18 @@ protected function setProperties(): void
             $value = $this->product->getManufacturer()->getMedia()->getUrl();
             $this->addProperty('vendorlogo', $value);
         }
+
+        if ($this->product->getPrice() && $this->product->getPrice()->first()) {
+            /** @var ProductPrice $price */
+            $price = $this->product->getPrice()->first();
```

Comment:
Is getting the first price safe to use? In which case can there be multiple prices and can we catch such a case as well?

Reply:
In general yes. Multiple prices are related to when there are prices for multiple currencies, so then each price would have a listing price accordingly (if any).

Reply:
Since we're using the Context of the Sales Channel when searching all products via DAL, does Shopware automatically give us only the price for the relevant Sales Channel? If not we need to fetch only those, which are actually relevant for the current Sales Channel. I want to avoid mixed up prices if customers provide multiple currency prices for different Sales Channels.